### DISCUSSION TITLE
470µF polymer capacitor

### DISCUSSION THREAD

**Discussion by Marc1850be** [jomjol/AI-on-the-edge-device#1952] [Q&A]
Hello,
Just a stupid question, the 470µF polymer capacitor, which voltage is it ? 
I found a lot of 25V / 16V does it need to be 5V ?
Thanks for your answer,
Best regards

**Comment by caco3** [comment]
Not sure which capacitor you refer to.
But the highest voltage on the board is 5V, so a 5V cap is ok. Electronics best practice is how ever to go a bit higher, eg. 6, 8 or 10V

**Comment by Slider0007** [comment]
@Marc1850be: 25V or 16V capacitors are also OK. It needs to be just greater than 5V (board voltage like already mentioned be caco3). The voltage which is mentioned is the maximum volatage rating of the capacitor.
